Data Collection Practices (#1191)
CocoaLumberjack/CocoaLumberjack: README.md

- for Xcode 6 or earlier, use the 2.0.x version
- for OS X < 10.7 support, use the 1.6.0 version

## Communication

- If you **need help**, use [Stack Overflow](http://stackoverflow.com/questions/tagged/lumberjack). (Tag 'lumberjack')
- If you'd like to **ask a general question**, use [Stack Overflow](http://stackoverflow.com/questions/tagged/lumberjack).
- If you **found a bug**, open an issue.
- If you **have a feature request**, open an issue.
- If you **want to contribute**, submit a pull request.

## Data Collection Practices

> Per  [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/), Apple is requesting app developers to provide info about their data collection, us SDK maintainers must provide them with the same data.

Reviewer: ```suggestion
Per [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/), Apple is requesting app developers to provide info about their data collection, us SDK maintainers must provide them with the same data.
```

Author: Done

### Data collection by the framework

**By default, CocoaLumberjack does NOT collect any data on its own.** 

[See our Data Collection Practices list.](https://cocoalumberjack.github.io/DataCollection/index.html)

### Indirect data collection through the framework

CocoaLumberjack is a logging framework which makes it easy to send those logs to different platforms.

> This is why collecting  data might happen quite easily, if the app developers include any sensitive data into the logs.

Reviewer: ```suggestion
This is why collecting data might happen quite easily, if app developers include any sensitive data in their log messages.
```

Author: Done

- If you **have a feature request**, open an issue.
- If you **want to contribute**, submit a pull request.

### Data Collection Practices

Per  [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/), Apple is requesting app developers to provide info about their data collection, us SDK maintainers must provide them with the same data.

#### Data collection by the framework

> **By default, CocoaLumberjack does not collect any data on its own.** 

Reviewer 2: Maybe it is better to emphasize this via quote? If you open "Display rich diff" view, you will find that two bold lines one below another will interfere and will hide second line.

Reviewer 2: Something like this

#### Data collection by the framework

```
 By default, CocoaLumberjack does NOT collect any data on its own.
```

Reviewer 2: Still not the best option, but, it separates header and body.

Author: I have converted all our headings from h3 to h2 (which was available), thus making "Data collection by the framework" an h3 which is much more distinct than an h4. Let me know if this is ok now

[See our updated list using the recommended format.](https://cocoalumberjack.github.io/DataCollection/index.html)

#### Indirect data collection through the framework

Considering CocoaLumberjack is a logging framework which makes it easy to send those logs to different platforms, collecting  data might happen quite easily, if the app developers include any sensitive data into those logs.

Private data includes but isn't limited to:

- user info (name, email, address, ...)
- location info
- contacts
- identifiers (user id, device id, ...)
- app usage data
- performance data
- health and fitness info
- financial info
- sensitive info
- user content
- history (browsing, search, ...)
- purchases
- diagnostics
- ...

Example: `DDLogInfo("User: \(myUser)")` will add the `myUser` info to the logs, so if those are forwarded to a 3rd party or sent via email, that may qualify as data collection.

> **The important thing is you app developers are fully responsible for adding sensitive data to the logs, thus documenting that via the Data Collection Practices info!**

Reviewer 2: Here, I would like to add section Important?

### Important note

```
App developers are fully responsible...
```

Author: Good idea, I have changed that am moved it up

Reviewer 2: I still not see it. Look at `rich diff view`.

If I read it in text, I won't spend enough attention/time on it. But I should.
**Important note: app developers are fully responsible for any sensitive data collected through our logging system!**
